static-analysis-plugin/plugin/src/test/groovy/com/novoda/test/LogsSubject.groovy

```diff
@@ -25,51 +27,56 @@ class LogsSubject extends Subject<LogsSubject, Logs> {
         Truth.assertAbout(FACTORY).that(logs)
     }
 
+    public static LogsSubject assumeThat(Logs logs) {
```

Comment:
well, i asked because everything being public, the usage of private would work as documentation. anyway, not a biggie.